devicetypes/stelpro/stelpro-maestro-thermostat.src/stelpro-maestro-thermostat.groovy

```diff
@@ -538,7 +552,7 @@ def configure() {
 	log.debug "binding to Thermostat cluster"
 	requests += zigbee.addBinding(THERMOSTAT_CLUSTER)
 	// Configure Thermostat Cluster
-	requests += zigbee.configureReporting(THERMOSTAT_CLUSTER, ATTRIBUTE_LOCAL_TEMP, DataType.INT16, 10, 60, 50)
+	requests += zigbee.configureReporting(THERMOSTAT_CLUSTER, ATTRIBUTE_LOCAL_TEMP, DataType.INT16, 1, 60, 50)
```

Comment:
10s was too frequent? This seems like a bad idea.

Reply:
This was a check to see if 10 was too infrequent. The issue, at least with this device, was that the Zigbee system was queuing up temperature reports from the thermostat system and then sending them to us about every 10 seconds. The firmware didn't seem to be clearing the queue if a new temperature report came in -- but the temperature alarm report would go to the front of the queue. So the result was we would get the alarm report, but then keep getting temperature reports up to 50C until the queue cleared, and the reports appeared to come at 10 second intervals. This doesn't seem to really help that.

I am going to revert this and just tweak the logic above.